squire_lib/src/operations/mod.rs

```diff
@@ -20,8 +20,9 @@ pub use player_ops::PlayerOp;
 #[derive(Serialize, Deserialize, Debug, Hash, Clone, PartialEq, Eq)]
 /// This enum captures all ways in which a tournament can mutate.
 pub enum TournOp {
-    /// Operation for a player register themself for a tournament
-    RegisterPlayer(SquireAccount),
+    /// Operation for a player register themself for a tournament, using a tournament-specific name
+    /// if their user name is already taken
+    RegisterPlayer(SquireAccount, Option<String>),
```

Comment:
You'll probably run into a number of issues with this change, and it's not really needed. After a player is registered, the caller can use the returned ID to get the player's name in the tournament.

Reply:
To be sure, is this meant for `OpData`?

Reply:
Oh ya, my bad. This was meant for `OpData`.